spec/truefi2/RateModel.spec

```diff
@@ -0,0 +1,20 @@
+methods {
+  poolBasicRate(address, uint256) returns uint256 envfree
```

Comment:
Nota bene for other reviewers (and myself when I forget :)).

`envfree` documentation: https://certora.atlassian.net/wiki/spaces/CPD/pages/181960777/Method+Declarations

Essentially, this function doesn't access `msg.sender`, `tx.origin`, `block.timestamp`, etc.
https://certora.atlassian.net/wiki/spaces/CPD/pages/7340101/Types